adhocracy4/dashboard/components/__init__.py

```diff
@@ -40,6 +40,7 @@ class DashboardComponent:
     identifier = ''
     weight = 0
     label = ''
+    for_superuser = False
```

Comment:
think the wording is fine and easy to understand, imho it would be a tiny little bit more accurate:
`for_superuser_only`, because `for_superuser = False` sounds like all other users are allowed, except of superusers.

Reply:
Yeah, I was also thinking about doing it the other way round and saying `for_initiators = True` or `initiators_allowed = True`. Yes, but I think your version is even clearer, so will do that after there is a second review. :)